public/app/popup/script.tsx

```
@@ -17,45 +21,70 @@ const origin = await $.connect("AppPopup");
+                            if (!(
+                                $.check(data.get("username") as string, $.SUG.PAT.Uname) &&
+                                $.check(data.get("password") as string, $.SUG.PAT.Uhashpass)
+                            )) {break}
+                            if (await $.curl("user/login", {
+                                Uname: data.get("username"),
+                                Uhashpass: data.get("password"),
+                            })) {
+                                window.location.reload();
+                            }
+                        }
+                        case false: {
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)